src/openrct2-ui/windows/LandRights.cpp

```diff
@@ -328,35 +328,35 @@ static void window_land_rights_tool_update_land_rights(ScreenCoordsXY screenCoor
     int16_t tool_length = (tool_size - 1) * 32;
 
     // Move to tool bottom left
-    mapTile.x -= (tool_size - 1) * 16;
-    mapTile.y -= (tool_size - 1) * 16;
-    mapTile.x &= 0xFFE0;
-    mapTile.y &= 0xFFE0;
+    mapTile->x -= (tool_size - 1) * 16;
+    mapTile->y -= (tool_size - 1) * 16;
+    mapTile->x &= 0xFFE0;
```

Comment:
This is such a common pattern that I think we should create a method for it. Called `Truncate` or something like that.

Not necessarily something for this PR, though.

Reply:
Agreed. Should be a method from CoordsXY. That and centre coordinate as there is tons of x+16, y+16

Reply:
Yeah, I though about that one too. I created https://github.com/OpenRCT2/OpenRCT2/issues/10298 for that.